test/test_core.py

```diff
@@ -1963,15 +1963,18 @@ def test_inlinejs4(self):
 
   def test_em_asm(self):
     self.maybe_closure()
+    self.set_setting('DEFAULT_LIBRARY_FUNCS_TO_INCLUDE', ['$UTF8ToString'])
```

Comment:
These days we can do this the C++ source code it self using `EM_JS_DEPS(deps, "$UTF8ToString")`